src/reducers/related-views.ts

```diff
@@ -0,0 +1,10 @@
+import {Action} from '../actions/index';
+import {DEFAULT_RELATED_VIEWS, RelatedViews} from '../models/related-views';
+
+export function relatedViewsReducer(relatedViewToggler: RelatedViews = DEFAULT_RELATED_VIEWS,
+                                    action: Action): RelatedViews {
+  const {isHidden} = relatedViewToggler;
+  return {
+    isHidden: !isHidden
```

Comment:
2) Why do we return `isHidden: !isHidden`, not just `isHidden`?

Reply:
I also wonder if just `hidden` is a better name than `isHidden`, but I don't have strong preference about this part.

Reply:
@kanitw Please correct me if I'm misunderstanding -- but I believe the `relatedViewToggler` parameter contains the current state before the action is pressed.  The value of `isHidden` is derived from `relatedViewToggler.isHidden` -- i.e. the current value of isHidden. `RelatedViewsAction` should flip the value of `isHidden` and returning `{isHidden}` as per your suggestion wouldn't flip the value at all.

@ssharif6 Is this what you were thinking?

Reply:
@felixcodes Yes. I was thinking along the lines of toggling the `isHidden` value. So a `isHidden: !isHidden` would flip from displaying hidden views in the state, and then not displaying them.

Reply:
That make sense. Still wouldn't you need an action?

Reply:
Yes, @ssharif6 is missing a 
switch (action.type) case RelatedViewsAction:

Reply:
case RELATED_VIEWS_HIDE_TOGGLE:*** (on mobile, can't edit comments)

Reply:
Yes! That is definitely true and is being added for next iteration